Readme.txt

```diff
@@ -1,77 +1,138 @@
                                 MORSE RUNNER
                               Contest Simulator
                                   freeware
-
-                   Version 1.80 Field Day Contest Prototype
+                                  
+                        Version 1.82 - ARRL DX Contest
+            The third release of the Morse Runner Community Edition
 
                Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
                       http://www.dxatlas.com/MorseRunner/
 
-        Copyright (C) 2022 Morse Runner Community Edition Contributors
+        Copyright (C) 2022-2023 Morse Runner Community Edition Contributors
```

Comment:
Should this (2023) wait till next year, assuming we publish the release this month?

Reply:
I pondered this too, I'll remove it.